integrations/azure-devops/.port/resources/port-app-config.yaml

```diff
@@ -1,77 +1,78 @@
-
+            project: .project.id | gsub(" "; "")
   - kind: repository-policy
     selector:
-      query: '.type.displayName=="Minimum number of reviewers"'
+      query: .type.displayName=="Minimum number of reviewers"
     port:
       entity:
         mappings:
-          identifier: '"\(.__repository.project.name | ascii_downcase | gsub(" "; ""))/\(.__repository.name | ascii_downcase | gsub(" "; ""))"'
+          identifier: >-
+            "\(.__repository.project.name | ascii_downcase | gsub("[ ();]"; ""))/\(.__repository.name | ascii_downcase | gsub("[ ();]"; ""))"
           blueprint: '"service"'
           properties:
-            minimumApproverCount: '.settings.minimumApproverCount'
-
+            minimumApproverCount: .settings.minimumApproverCount
   - kind: repository-policy
     selector:
-      query: '.type.displayName=="Work item linking"'
+      query: .type.displayName=="Work item linking"
     port:
       entity:
         mappings:
-          identifier: '"\(.__repository.project.name | ascii_downcase | gsub(" "; ""))/\(.__repository.name | ascii_downcase | gsub(" "; ""))"'
+          identifier: >-
+            "\(.__repository.project.name | ascii_downcase | gsub("[ ();]"; ""))/\(.__repository.name | ascii_downcase | gsub("[ ();]"; ""))"
           blueprint: '"service"'
           properties:
-            workItemLinking: '.isEnabled and .isBlocking'
+            workItemLinking: .isEnabled and .isBlocking
   - kind: work-item
     selector:
       query: 'true'
     port:
       entity:
         mappings:
-          identifier: '.id | tostring'
+          identifier: .id | tostring
+          title: .fields."System.Title"
           blueprint: '"workItem"'
-          title: '.fields."System.Title"'
           properties:
-            type: '.fields."System.WorkItemType"'
-            state: '.fields."System.State"'
-            effort: '.fields."Microsoft.VSTS.Scheduling.Effort"'
-            description: '.fields."System.Description"'
-            link: '.url'
-            reason: '.fields."System.Reason"'
-            createdBy: '.fields."System.CreatedBy".displayName'
-            changedBy: '.fields."System.ChangedBy".displayName'
-            createdDate: '.fields."System.CreatedDate"'
-            changedDate: '.fields."System.ChangedDate"'
+            type: .fields."System.WorkItemType"
+            state: .fields."System.State"
+            effort: .fields."Microsoft.VSTS.Scheduling.Effort"
+            description: .fields."System.Description"
+            link: .url
+            reason: .fields."System.Reason"
+            createdBy: .fields."System.CreatedBy".displayName
+            changedBy: .fields."System.ChangedBy".displayName
+            createdDate: .fields."System.CreatedDate"
+            changedDate: .fields."System.ChangedDate"
           relations:
-            project: '.fields."System.TeamProject" | gsub(" "; "")'
+            project: .__projectId
```

Comment:
This should be also updated to be without special characters?

Reply:
good catch